pkg/monitoring/domainstats/downwardmetrics/scraper.go

```diff
@@ -58,19 +83,16 @@ func (s *Scraper) Scrape(socketFile string, vmi *k6sv1.VirtualMachineInstance) {
 			log.Log.Reason(err).Warning("getQemuVersion not implemented, consider to upgrade kubevirt")
 			version = qemuVersionUnknown
 		} else {
-			log.Log.Reason(err).Errorf("failed to update qemu stats from socket %s", socketFile)
-			return
+			return nil, fmt.Errorf("failed to update qemu stats from socket %s", socketFile)
 		}
 	}
 
 	vmStats, exists, err := cli.GetDomainStats()
 	if err != nil {
-		log.Log.Reason(err).Errorf("failed to update stats from socket %s", socketFile)
-		return
+		return nil, fmt.Errorf("failed to update stats from socket %s", socketFile)
```

Comment:
as above, We are losing the `err` content duing this
```suggestion
        return nil, fmt.Errorf("failed to update stats from socket %s: %s", socketFile, err.Error())
```

Reply:
done